docs/konflux-onboarding.md

```diff
@@ -141,6 +141,9 @@ If you plan to use your catalogs on multiple cluster architectures, you will nee
 
 A sample PR for these changes can be found in https://github.com/konflux-ci/olm-operator-konflux-sample/pull/65.
 
+### Storing the FBC fragments built
+By default once the FBC fragments are built, the images will be stored in quay.io/redhat-user-workloads namespace. In order to release those FBC fragments to OCP index images using IIB, the images can be public or private within quay.io/redhat-user-workloads. If you intend to use any other namespace, the repos should be public, or IIB should have access to it during the release.
```

Comment:
product teams might not know what IIB is or how it fits in the FBC pipeline. I would recommend avoiding using the term IIB. Consider something like...

```
FBC fragments are stored in the `redhat-user-workloads` Quay namespace by default. The visibility of the Quay repo can be private as long as you are using the default repository. If you decide to house them elsewhere in a private organization, it is mandatory for the repo to be **public** at the time of the release. 
```